package.json

```diff
@@ -1,232 +1,143 @@
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!src/**/*.d.ts"
-    ],
-    "setupFiles": [
-      "react-app-polyfill/jsdom"
-    ],
-    "setupFilesAfterEnv": [
-      "<rootDir>/src/setupTests.ts"
-    ],
-    "testMatch": [
-      "<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
-      "<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}"
-    ],
-    "testEnvironment": "jsdom",
-    "transform": {
-      "^.+\\.(js|jsx|mjs|cjs|ts|tsx)$": "<rootDir>/config/jest/babelTransform.js",
-      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
-      "^(?!.*\\.(js|jsx|mjs|cjs|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
-    },
-    "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|cjs|ts|tsx)$",
-      "^.+\\.module\\.(css|sass|scss)$"
-    ],
-    "modulePaths": [],
-    "moduleNameMapper": {
-      "^react-native$": "react-native-web",
-      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
-    },
-    "moduleFileExtensions": [
-      "web.js",
-      "js",
-      "web.ts",
-      "ts",
-      "web.tsx",
-      "tsx",
-      "json",
-      "web.jsx",
-      "jsx",
-      "node"
-    ],
-    "watchPlugins": [
-      "jest-watch-typeahead/filename",
-      "jest-watch-typeahead/testname"
-    ],
-    "resetMocks": true
-  },
-  "babel": {
-    "presets": [
-      "react-app"
-    ]
-  }
+	"name": "front-end",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@axa-fr/react-oidc": "^7.22.6",
+		"@fontsource/source-sans-pro": "^5.0.8",
+		"@heroicons/react": "^2.1.3",
+		"@pdc/sdk": "^0.11.1",
+		"dayjs": "^1.11.11",
+		"jsonpath-plus": "^9.0.0",
+		"pino": "^9.3.1",
+		"pretty-bytes": "^6.1.1",
+		"react": "^18.3.1",
+		"react-dom": "^18.3.1",
+		"react-router-dom": "^6.23.1",
+		"title-case": "^4.3.1",
+		"web-vitals": "^4.0.1"
+	},
+	"devDependencies": {
+		"@babel/plugin-proposal-private-property-in-object": "^7.21.11",
+		"@babel/preset-typescript": "^7.24.7",
+		"@storybook/addon-essentials": "^8.1.3",
+		"@storybook/addon-interactions": "^8.1.3",
+		"@storybook/addon-links": "^8.1.3",
+		"@storybook/blocks": "^8.0.5",
+		"@storybook/react": "^8.1.3",
+		"@storybook/react-vite": "8.1.3",
+		"@storybook/test": "^8.0.5",
+		"@storybook/react-webpack5": "^8.0.9",
+		"@testing-library/react": "^15.0.7",
+		"@testing-library/user-event": "^14.5.2",
+		"@types/node": "^18.19.39",
+		"@types/react": "^18.3.3",
+		"@types/react-dom": "^18.3.0",
+		"@typescript-eslint/eslint-plugin": "^7.17.0",
+		"@typescript-eslint/parser": "^7.17.0",
+		"@vitejs/plugin-react": "^4.3.1",
+		"babel-plugin-named-exports-order": "^0.0.2",
+		"eslint": "^v8.57.0",
+		"eslint-config-airbnb": "^19.0.4",
+		"eslint-config-airbnb-typescript": "^18.0.0",
+		"eslint-config-prettier": "^9.1.0",
+		"eslint-plugin-react-hooks": "^4.6.2",
+		"eslint-plugin-react-refresh": "^0.4.7",
+		"jsdom": "^24.0.0",
+		"prettier": "^3.3.3",
+		"storybook": "^8.1.3",
+		"typescript": "^5.5.4",
```

Comment:
No there's no requirement to upgrade here, I think since part of the reason we wanted to migrate was to upgrade typescript (as it was blocked by CRA) I included it here, but it can be a separate version bump!

Reply:
Just to double clarify: the real answer to the question Jason asked is "yes!"

We could avoid updating typescript, but that would basically mean migrating to an older version of vite, and then migrating to a newer version of vite separately -- not worth the increase in effort, and we can just think of the TypeScript bump as part of the vite migration path.